modules/ossm-cr-example.adoc

```diff
@@ -18,63 +18,257 @@ This example `ServiceMeshControlPlane` definition contains all of the supported
-        ior_enabled: false
+  gateways:
+    ingress: # _the_ istio-ingressgateway
+      service:
+        type: ClusterIP
+        ports:
+        - name: status-port
+          port: 15020
+        - name: http2
+          port: 80
+          targetPort: 8080
+        - name: https
+          port: 443
+          targetPort: 8443
+      meshExpansionPorts: []
+    egress: # _the_ istio-egressgateway
+      service:
+        type: ClusterIP
+        ports:
+        - name: status-port
+          port: 15020
+        - name: http2
+          port: 80
+          targetPort: 8080
+        - name: https
+          port: 443
+          targetPort: 8443
+    additionalIngress:
+      some-other-ingress-gateway: {}
+    additionalEgress:
+      some-other-egress-gateway: {}
 
+  policy:
     mixer:
-      policy:
-        autoscaleEnabled: false
+      enableChecks: false # default
+      failOpen: false # default
+      runtime:
+        container:
+          image: mixer # default mixer image
+          resources: {} # mixer resources
 
-      telemetry:
-        autoscaleEnabled: false
-        resources:
-          requests:
-            cpu: 100m
-            memory: 1G
-          limits:
-            cpu: 500m
-            memory: 4G
+  telemetry:
+    type: <istiod_#or_Mixer_or_Remote>
+    mixer: # legacy v1
+      sessionAffinity: false
+      batching:
+        maxEntries: 100
+        maxTime: 1s
+      adapters:
+        useAdapterCRDs: false
+        kubernetesenv: true
+        stdio:
+          outputAsJSON: false
+        prometheus:
+          metricsExpiryDuration: 10m
+        stackdriver: null
 
+  runtime:
     pilot:
-      autoscaleEnabled: false
-      traceSampling: 100
+      deployment:
+        replicas: 2
+      pod:
+        affinity: {}
+      container:
+        resources:
+          limits: {}
+          requirements: {}
 
-    kiali:
-      enabled: true
+  addons:
+    visualization:
+      grafana:
+        address: some-grafana-url 
+        install: 
+          selfManaged: true 
+          config:
+            env: {} 
+            envSecrets: {} 
+          persistence:
+            storageClassName: ""
+            accessMode: ReadWriteOnce
+            capacity: 5Gi 
+          service: 
+            ingress:
+              contextPath: /grafana
+              tls:
+                termination: reencrypt
+          runtime:
+            deployment: {}
+            pod: {}
+      kiali:
+        name: kiali 
+        install: # install kiali CR if not present
+          config:
+            dashboard:
+              viewOnly: false
+              enableGrafana: true
+              enableTracing: true
+              enablePrometheus: true
+          service: 
+            ingress:
+              contextPath: /kiali
+          runtime:
+            deployment: {}
+            pod: {}
+----
 
-    grafana:
-      enabled: true
+The following table lists the parameters for the `ServiceMeshControlPlane` resource.
```

Comment:
Why did the Jaeger configuration disappear?  All of the examples that I found on GitHub still include Jaeger configuration.

https://github.com/maistra/istio-operator/blob/maistra-2.0/deploy/examples/maistra_v2_servicemeshcontrolplane_cr_auth.yaml

https://github.com/maistra/istio-operator/blob/maistra-2.0/deploy/examples/maistra_v2_servicemeshcontrolplane_cr_full.yaml

https://github.com/maistra/istio-operator/blob/maistra-2.0/deploy/examples/maistra_v2_servicemeshcontrolplane_cr_minimal.yaml

Reply:
For reference/completeness, but note that Jaeger is enabled by setting tracing.type=Jaeger.  This just configures the jaeger installation:
```yaml
    jaeger:
      name: jaeger
      install:
        storage:
          type: Memory # or Elasticsearch
          memory:
            maxTraces: 100000
          elasticsearch:
            nodeCount: 3
            storage: {} # jaeger elasticsearch storage configuration
            redundancyPolicy: SingleRedundancy
            indexCleaner: {} # jaeger elasticsearch esIndexCleaner configuration
        ingress: {} # jaeger ingress configuration
```